### ISSUE TITLE
Test test_generate_json_schema fail on v0.2.1

### ISSUE THREAD

**Issue by GaetanLepage** [astral-sh/ruff#9923]
The following test fail on the latest release:
```
failures:

---- generate_json_schema::tests::test_generate_json_schema stdout ----
Error: ruff.schema.json changed, please run `cargo dev generate-all`:
```

This is the part of the diff which differs:
```
         "RUF2",
         "RUF20",
         "RUF200",
>        "RUF9",
>        "RUF90",
>        "RUF900",
>        "RUF901",
>        "RUF902",
>        "RUF903",
>        "RUF91",
>        "RUF911",
>        "RUF912",
>        "RUF92",
>        "RUF920",
>        "RUF921",
>        "RUF95",
>        "RUF950",
         "S",
         "S1",
         "S10",
         "S101",
         "S102",
```

**Comment by GaetanLepage** [comment]
Seems to work fine on `main` though.

**Comment by charliermarsh** [comment]
Yup, I believe this was fixed in #9901. Thanks!
